core/src/main/java/brooklyn/entity/rebind/RebindManagerImpl.java

```diff
@@ -361,6 +364,38 @@ public void stop() {
         if (persistenceStoreAccess != null) persistenceStoreAccess.stop(true);
     }
     
+        
+    public void rebindPartialActive(CompoundTransformer transformer, Iterator<BrooklynObject> objectsToRebind) {
+        final ClassLoader classLoader = 
+            managementContext.getCatalog().getRootClassLoader();
+        final RebindExceptionHandler exceptionHandler = 
+            RebindExceptionHandlerImpl.builder()
+                .danglingRefFailureMode(danglingRefFailureMode)
```

Comment:
have added a TODO

the update-proxy-then-unmanage-old is mostly atomic so most failures should be pretty safe, but i'm sure there will be edge conditions

when we better understand the failure dimensions (and have visualization) then we can tidy this up